src/main/java/com/synopsys/integration/detect/tool/binaryscanner/BinaryUploadOperation.java

```diff
@@ -66,6 +66,26 @@ public CodeLocationCreationData<BinaryScanBatchOutput> uploadBinaryScanFile(
             throw new DetectUserFriendlyException("Failed to upload binary scan file.", e, ExitCodeType.FAILURE_BLACKDUCK_FEATURE_ERROR);
         }
     }
+    
+    public CodeLocationCreationData<BinaryScanBatchOutput> uploadBinaryScanFiles(
```

Comment:
Confused as the need for this method. Just trying to understand as we'll need to port this to the newer approach I'm working on.

Reply:
It is not needed and is not in use. I wrote that quite some time ago. We can remove it unless you want to repurpose it/move it.

Reply:
It is removed. Thanks!